src/App.tsx

```diff
@@ -1,26 +1,72 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useMemo, useEffect, useState } from 'react';
+import { getTodos } from './api';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { todosSlice } from './features/todos';
+import { Status } from './types/Status';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
+  const todo = useAppSelector(state => state.currentTodo);
+  const todos = useAppSelector(state => state.todos);
+  const dispatch = useAppDispatch();
+  const [isLoading, setIsLoading] = useState(false);
+  const filteres = useAppSelector(state => state.filter);
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  const filteredTodos = useMemo(() => {
```

Comment:
You can move the function outside the file, and then use it as a helper func in this place.